src/Fantomas/Program.fs

```diff
@@ -64,9 +55,9 @@ type OutputPath =
 
 [<RequireQualifiedAccess>]
 type ProcessResult =
-    | Formatted of string
+    | Formatted of string * ProfileInfos option
```

Comment:
Shower thought: do we need this difference between `FormatResult` and `ProcessResult`?
Perhaps we can just use `FormatResult` instead?

Reply:
Yep, seems like we can.